conf/local.yml

```diff
@@ -1,9 +1,9 @@
 ---
 
-host: 127.0.0.1
+host: 0.0.0.0
 port: 8888
 exfil_dir: /tmp
-memory: True
+memory: False
```

Comment:
what is motivating the change here?

Reply:
More debug that somehow got stuffed into this.